experimental/gcp-log/function.go

```diff
@@ -23,53 +23,89 @@ import (
 	"fmt"
 	"net/http"
 	"os"
-
-	"github.com/transparency-dev/merkle/rfc6962"
-	"golang.org/x/mod/sumdb/note"
-	"google.golang.org/api/iterator"
+	"path/filepath"
 
 	"github.com/gcp_serverless_module/internal/storage"
-	"github.com/transparency-dev/serverless-log/pkg/log"
 
+	"cloud.google.com/go/kms/apiv1"
+	"github.com/transparency-dev/armored-witness/pkg/kmssigner"
```

Comment:
Should be pop in a TODO to move that package to somewhere else to be reused?

Reply:
Yes, let me do it in the `armored-witness` repo! https://github.com/transparency-dev/armored-witness/pull/56